apps/block_scout_web/assets/js/lib/datepicker.js

```diff
@@ -44,11 +45,12 @@ $button.on('click', () => {
     })
       .done(function (data) {
         // eslint-disable-next-line
-        grecaptcha.reset()
+        grecaptcha.enterprise.reset()
         const dataJson = JSON.parse(data)
-        if (dataJson.success) {
+        if (dataJson.tokenProperties.valid === true) {
           $button.removeClass('spinner')
           location.href = $button.data('link')
+          $message.removeClass('hidden')
```

Comment:
Are you adding that class again when the csv is ready? I wasn't able to find that.

Reply:
No, I'd expect the user to navigate away after that. That's a good idea though, I'll look into adding it in another patch